src/components/Form.vue

```diff
@@ -30,15 +37,11 @@ interface FormInterface {
     },
 })
 export default class Form extends Vue {
-    @Prop({type: Object, default: () => ({}) })
+    @Prop({default: () => ({}) })
     readonly source_form!: FormInterface
-    @Prop(String)
+    @Prop({required: true})
     readonly send_url!: string
-    @Prop({type: String, default: 'get'})
+    @Prop({default: 'get'})
```

Comment:
I'd say that `POST` is definitely more common method for forms. I'd suggest using it as a default.

Reply:
Fixed